elements/rh-secondary-nav/demo/rh-secondary-nav.html

```diff
@@ -3,168 +3,157 @@
 <script type="module" src="rh-secondary-nav.js"></script>
 
 <div class="mock-nav"></div>
-<rh-secondary-nav role="navigation">
+<rh-secondary-nav role="navigation" data-analytics-region="secondary nav">
```

Comment:
Since this demo is using Ansible as the example, can we make this *-region="Ansible secondary nav" ?

Reply:
Good call!